src/View/AbstractDirectoryView.vala

```diff
@@ -378,8 +402,7 @@ namespace Files {
         }
 
         private void set_up_directory_view () {
-            set_policy (Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC);
-            set_shadow_type (Gtk.ShadowType.NONE);
+            scrolled_window.set_policy (Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC);
```

Comment:
Can we just set hscrollpolicy here since vscroll is automatic by default and we're only changing the hscroll?